pkg/scheduler/core/generic_scheduler_test.go

```diff
@@ -2258,3 +2258,42 @@ func assignDefaultStartTime(pods []*v1.Pod) {
 		}
 	}
 }
+
+func TestFairEvaluationForNodes(t *testing.T) {
+	defer algorithmpredicates.SetPredicatesOrderingDuringTest(order)()
+	predicates := map[string]algorithmpredicates.FitPredicate{"true": truePredicate}
+	numAllNodes := 500
+	nodeNames := make([]string, 0, numAllNodes)
+	for i := 0; i < numAllNodes; i++ {
+		nodeNames = append(nodeNames, strconv.Itoa(i))
+	}
+	nodes := makeNodeList(nodeNames)
+	g := makeScheduler(predicates, nodes)
+	g.percentageOfNodesToScore = 30
```

Comment:
any reason not to keep the default?

Reply:
@mrkm4ntr ?

Reply:
Default is 50. I'd like to make `numAllNodes mod percentageOfNodesToScore != 0`. It is ok to change numAllNodes to odd and percentageOfNodesToScore to default, but is this so important?

Reply:
add a comment.